pyasdf/asdf.py

```diff
@@ -406,6 +365,54 @@ def open(cls, fd, uri=None, mode='r',
 
         return self
 
+    @classmethod
+    def open(cls, fd, uri=None, mode='r',
+             validate_checksums=False,
+             extensions=None,
+             do_not_fill_defaults=False,
+             _get_yaml_content=False):
```

Comment:
That's a good point.  It's only there to make writing some sorts of tests possible.